### PR TITLE
Added the fees for L1<->L2 messages

### PR DESCRIPTION
**Pull request by qbzzt**
Added a page about the gas fees for messages from L1 to L2 (which are typically just the L1 cost, but if the message requires too much L2 gas might have to include that), and for messages from L2 to L1 (which include both the initial transaction on L2, and the finalization transaction on L1).

### CODE REVIEW

File: src/docs/developers/bridge/fees.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+::: tip OVM 2.0 Release Dates
+OVM 2.0 is already released on the Kovan test network.
+We expect to deploy it to the production Optimistic Ethereum network on November 11th.
+:::
+
+## Fees for L1 to L2 transactions
+
+For an L1 to L2 transaction the major fee is the L1 cost of submitting the transaction. You send a transaction to the [`L1CrossDomainMessenger`](https://github.com/ethereum-optimism/optimism/blob/regenesis/0.5.0/packages/contracts/contracts/L1/messaging/L1CrossDomainMessenger.sol)
```
Comment (Nicca42) [inline src/docs/developers/bridge/fees.md:13]: ...the majority of the cost is the L1 fee for submitting the transaction

File: src/docs/developers/bridge/fees.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: The Fees for Sending Data Between L1 and L2 (in OVM 2.0)
+lang: en-US
+---
+
+::: tip OVM 2.0 Release Dates
+OVM 2.0 is already released on the Kovan test network.
+We expect to deploy it to the production Optimistic Ethereum network on November 11th.
+:::
```
Comment (Nicca42) [inline src/docs/developers/bridge/fees.md:9]: Remove this tip.

File: src/docs/developers/bridge/fees.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Fees for L1 to L2 transactions
+
+For an L1 to L2 transaction the major fee is the L1 cost of submitting the transaction. You send a transaction to the [`L1CrossDomainMessenger`](https://github.com/ethereum-optimism/optimism/blob/regenesis/0.5.0/packages/contracts/contracts/L1/messaging/L1CrossDomainMessenger.sol)
+contract, which then sends a call to the [`CanonicalTransactionChain`](https://github.com/ethereum-optimism/optimism/blob/regenesis/0.5.0/packages/contracts/contracts/L1/rollup/CanonicalTransactionChain.sol). This generally isn't *too* expensive, but it mainly depends on L1 gas costs.
```
Comment (Nicca42) [inline src/docs/developers/bridge/fees.md:14]: ...but is ultimately determined by L1 gas costs.

File: src/docs/developers/bridge/fees.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+An L1 to L2 message can trigger contract execution on L2. The gas limit for that transaction is provided as part of the message. If the gas limit is below 1.92 million (at writing, it might change) that L2 gas is free. For the gas above that limit we burn one L1 gas for every 32 L2 gas to prevent denial of service attacks.
+
+
+## Fees for L2 to L1 transactions
+
+Each message from L2 to L1 requires two transactions:
+
+1. An L2 transaction that *initiates* the transaction, which is priced the same way that Sequencer transactions are priced.
+1. An L1 transaction that *finalizes* the transaction, after the verification challenge period (one week on mainnet) has passed. This transaction is somewhat expensive because it includes verifying a [Merkle trie](https://eth.wiki/fundamentals/patricia-tree) inclusion proof.
```
Comment (Nicca42) [inline src/docs/developers/bridge/fees.md:26]: An L1 transaction that *finalizes* the transaction. This transaction can only be submitted after the verification challenge period (one week on mainnet) has passed.

Comment (Nicca42) [reply src/docs/developers/bridge/fees.md:26]: I don't like the way we are explaining the challenge period, but I cannot think of a better way. I want it to be something like: one week from initialisation, but I can't figure out how to squeeze that into the sentence. Might need to be broken up more.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/developers/bridge/fees.md (hunk 4 above)
Comment (qbzzt) [reply src/docs/developers/bridge/fees.md]: What about now? I just linked to where we explain it.

File: src/.vuepress/config.js [hunk 5]
```diff
@@ -88,7 +88,8 @@ module.exports = {
                 // Common (standard bridge) before
                 // rare (messaging)
                 '/docs/developers/bridge/standard-bridge.md',
-                '/docs/developers/bridge/messaging.md'
+                '/docs/developers/bridge/messaging.md',
+                '/docs/developers/bridge/fees.md'
```
Comment (elenadimitrova) [inline src/.vuepress/config.js:92]: Can we rename this page to `cross-domain-fees` as we had a single `fees` page before whereas now we have this plus `transaction-fees` under dev docs.

**Review by elenadimitrova — APPROVED** [review]

**Pull request merged**